### PR TITLE
Use `just` instead of `meson`, and don't depend on GTK

### PR DESCRIPTION
**Pull request by ids1024**
Based on packaging for `cosmic-launcher`. We don't want to depend on GTK and the `gnome` meson module if we're not using GTK. Though we have a couple things using `just` and other using `make`, so we still need to standardize exactly how we're doing builds.

Nix flake needs to be updated, but it looks like the flake for `cosmic-laucher` also doesn't work currently, and I'm not familiar enough with nix.

### CODE REVIEW

**Comment by ids1024** [comment]
Cargo fmt issue is unrelated to this PR.

I forgot about the Flatpak. Not sure if it makes sense to have Flatpak packages for shell components like this, though it may still be something worth experimenting with.

**Review by wash2 — APPROVED** [review]
Ya I think for our shell components it probably doesn't make much sense to use flatpaks for shell components, but I can see how it might be good to encourage unofficial COSMIC applications (or applets for the panel) to be flatpaks.

File: po/meson.build [hunk 1]
```diff
@@ -1 +0,0 @@
-i18n.gettext(gettext_package, preset: 'glib')
```
Comment (eli-schwartz) [inline po/meson.build:1]: This doesn't seem to be reimplemented but on the other hand it seems like the project never actually went and created those translations anyway?

File: meson.build [hunk 2]
```diff
@@ -1,76 +0,0 @@
...
-
-subdir('data')
-subdir('po')
-subdir('src')
-
-gnome.post_install(
-  gtk_update_icon_cache: true,
-  glib_compile_schemas: true,
-  update_desktop_database: true,
-)
```
Comment (eli-schwartz) [inline meson.build:76]: This doesn't seem to be reimplemented in the justfile, which means that icon themes and desktop databases will be wrong for the user by default.

I don't see a gschema ether way, so perhaps that shouldn't have ever been there to begin with?

File: po/meson.build (hunk 1 above)
Comment (ids1024) [reply po/meson.build:1]: Yes, this was based on the gtk-rs meson template, so it wasn't using these things anyway.

Comment (eli-schwartz) [reply po/meson.build:1]: In that case, you might want to remove the unused po/ directory altogether?

**Pull request merged**